components/vf-tree/vf-tree.njk

```diff
@@ -1,49 +1,31 @@
-<div class="vf-tree{% if collapsed %} vf-tree--collapsed{% endif %}" {% if collapsed %}data-vf-tree--collapsed="true"{% endif %}>
+{% set vfTreeButton %}
+<button href="{{ tree_example_href }}" class="vf-button vf-tree__button">
+  <svg width="8" height="13" xmlns="http://www.w3.org/2000/svg"><g stroke="" stroke-width="1.6" fill="none" fill-rule="evenodd" stroke-linecap="square"><path d="M1.417 1.275L6 6.326M6 6.105l-4.583 5.61"/></g></svg>
+</button>
+{% endset %}
+
+{% macro vfTreeList(list, currentDepth=1) %}
```

Comment:
I thought we weren't using `macro` - but that could have been the use case I had a while back (moving 'property logic' out of the `vf-button.njk` file :sob:

Reply:
Is there a downside to using macro?

Reply:
I think it was more me having a separate file to abstract the logic more than using it.

Reply:
Cool. To me this seems a good place to use a macro, given the repeated amount of code ... and in principle we could now support depths of 20 or more! 🙈